tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "useDefineForClassFields": true,
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
     "module": "ESNext",
-    "skipLibCheck": false,
```

Comment:
Did you need to change this for some reason?

Reply:
Yep--`yarn tsc` was failing on errors in node_modules, and we don't really need to type check those anyway